### ISSUE TITLE
this.gridster.previewStyle is not a function

### ISSUE THREAD

**Issue by rwpino** [tiberiuzuld/angular-gridster2#219]
Hello  I m getting this error when I change from one route to another
```
"Error: Uncaught (in promise): TypeError: this.gridster.previewStyle is not a function
TypeError: this.gridster.previewStyle is not a function
    at GridsterDraggable.destroy (webpack-internal:///../../../../angular-gridster2/dist/gridsterDraggable.service.js:22:23)
    at GridsterItemComponent.ngOnDestroy (webpack-internal:///../../../../angular-gridster2/dist/gridsterItem.component.js:46:19)
    at callProviderLifecycles (webpack-internal:///../../../core/esm5/core.js:12912:18)
    at callElementProvidersLifecycles (webpack-internal:///../../../core/esm5/core.js:12873:13)
    at callLifecycleHooksChildrenFirst (webpack-internal:///../../../core/esm5/core.js:12856:29)
    at destroyView (webpack-internal:///../../../core/esm5/core.js:14222:5)
    at callViewAction (webpack-internal:///../../../core/esm5/core.js:14373:13)
    at execComponentViewsAction (webpack-internal:///../../../core/esm5/core.js:14285:13)
    at destroyView (webpack-internal:///../../../core/esm5/core.js:14221:5)
    at callWithDebugContext (webpack-internal:///../../../core/esm5/core.js:15256:42)
    at resolvePromise (webpack-internal:///../../../../zone.js/dist/zone.js:809:31)
    at resolvePromise (webpack-internal:///../../../../zone.js/dist/zone.js:775:17)
    at eval (webpack-internal:///../../../../zone.js/dist/zone.js:858:17)
    at ZoneDelegate.invokeTask (webpack-internal:///../../../../zone.js/dist/zone.js:421:31)
    at Object.onInvokeTask (webpack-internal:///../../../core/esm5/core.js:4939:33)
    at ZoneDelegate.invokeTask (webpack-internal:///../../../../zone.js/dist/zone.js:420:36)
    at Zone.runTask (webpack-internal:///../../../../zone.js/dist/zone.js:188:47)
    at drainMicroTaskQueue (webpack-internal:///../../../../zone.js/dist/zone.js:595:35)
    at <anonymous>"
```
is coming from ngOnDestroy 
any ideas why this is happening ?
thanks in advance

**Comment by tiberiuzuld** [comment]
Hi @rwpino ,
The `previewStyle` function is destroyed before items so is removed from `this.gridster`.
Will have a fix shortly.

Thanks

**Comment by tiberiuzuld** [comment]
Ok now should be fixed in v4.4.1

Thanks

**Comment by rwpino** [comment]
thanks!
